### DISCUSSION TITLE
Game Crashing when i load a character

### DISCUSSION THREAD

**Discussion by ValenG2002** [KadeArchive/Kade-Engine#1689] [Q&A]
hello, I added a character in the code of the kade engine, but when loading it with the debug the game crashes, I put the code correctly and it still remains the same, is there something missing?
i'm using de Kade Engine 1.5.2
![Codigo](https://user-images.githubusercontent.com/86801701/128809265-edf7746e-c0e1-473c-a94f-4996236b847c.PNG)

![image](https://user-images.githubusercontent.com/86801701/128809313-8d9f3d7d-0dc0-4034-96c5-fc3ca8e2c98e.png)

**Comment by raf13lol** [comment]
have you added icon to icon grid?

**Reply by ValenG2002** [reply]
in the second image it is seen that it is, and the icon in the debug is shown

**Reply by raf13lol** [reply]
send xml and png please

**Reply by ValenG2002** [reply]
![gura_amelia](https://user-images.githubusercontent.com/86801701/128904351-fb28e5a5-8c4f-4fc0-8033-3e23d148f88c.png)
XML: https://www.mediafire.com/file/9y1kwp8op4umwue/gura_amelia.xml/file
Here, the characters are from Holofunk mod

**Reply by raf13lol** [reply]
where is the code for misses?

**Reply by raf13lol** [reply]
send character.hx please

**Reply by raf13lol** [reply]
send HealthIcon.hx please

**Reply by raf13lol** [reply]
icon grid with its xml please

**Reply by raf13lol** [reply]
i will try my best to help your issue

**Reply by raf13lol** [reply]
send healthicon.hx please sorry

**Reply by ValenG2002** [reply]
i put the misses too, but the problem is still showing.
biu well, i'm going
Healthicon: https://www.mediafire.com/file/uj0ajrxmp0ez018/HealthIcon.hx/file
Character: https://www.mediafire.com/file/ggcccff4go135s3/Character.hx/file
![iconGrid](https://user-images.githubusercontent.com/86801701/128906539-3182d418-a040-4ac3-8980-fbda3cd969fc.png)
Yeah, i tried with Sky too, but it's doesn't work

**Reply by raf13lol** [reply]
sorry you didnt send icongrid.xml

**Reply by ValenG2002** [reply]
There is no file with that name in my Kade engine.

**Reply by raf13lol** [reply]
im sorry i cant seem to help but try  this character.hx just change the .txt to .hx
[Character.txt](https://github.com/KadeDev/Kade-Engine/files/6963404/Character.txt)
